cloudwash/providers/aws.py

```diff
@@ -60,13 +60,33 @@ def dry_pips():
                     [dry_data["PIPS"]["delete"].append(dpip["AllocationId"]) for dpip in rpips]
                 return rpips
 
+            def dry_stacks():
+                rstacks = []
+                [
+                    rstacks.append(stack.name)
+                    for stack in aws_client.list_stacks()
+                    if (
+                        total_running_time(stack).minutes
+                        >= settings.aws.criteria.stacks.sla_minutes
+                        or stack.name.startswith(settings.aws.criteria.stacks.delete_stack)
```

Comment:
This has to be `and` as per our `dry_vms` implementation as to match both sla_minutes and prepend text match.

```suggestion
                        and stack.name.startswith(settings.aws.criteria.stacks.delete_stack)
```